apps/server/src/api-data/rundown/rundown.controller.ts

```diff
@@ -69,6 +70,21 @@ export async function rundownBatchPut(req: Request, res: Response<MessageRespons
   }
 }
 
+export async function rundownFrozenPost(req: Request, res: Response<MessageResponse | ErrorResponse>) {
+  if (failEmptyObjects(req.body, res)) {
+    return res.status(404);
+  }
```

Comment:
question: would this not have been taken care of by the validator?

Reply:
Most of the other functions here had that check at the top so I just followed suit.

Reply:
ah I see.
There is a case where the other code is also wrong,

But in some cases, all the elements of the body are optional, so the validation could potentially pass with an empty object

Reply:
Fair. I've cleaned that out and this should be good to go now.